docs/versioned/admin/admin-overview.md

```diff
@@ -0,0 +1,125 @@
+---
+audience: administrator
+components:
+  - serving
+  - eventing
+function: reference
+---
+# Overview
+
+Knative has a set of tools and capabilities to administer your Kubernetes clusters. This article provides a overview of Knative features, capabilities, and resources of interest to Kubernetes Administrators, and is organized by the following areas:
+
+- Installation
+- Configuration
+- Monitoring and Observability
+- Security and Access Control
+- Updates and Maintenance
+
+## Installation
+
+Use the Installation Roadmap for guidance on installing Knative using the resources and tool choices that best fit your needs. You can install using YAML resources, manifests, or CLI tools that include the Knative Operator.
```

Comment:
```suggestion
Use the [Installation Roadmap](../install/README.md) for guidance on installing Knative using the resources and tool choices that best fit your needs. You can install using YAML manifests or a Kubernetes operator; the operator supports management via the `kn` CLI.
```

"manifests" is a term for resources declared in YAML format, so those two sort of mean the same thing.

Reply:
It may be worth explaining that Knative can add support for new transports after the initial installation, so your initial choices don't lock you in long-term (though migrating from one message transport or network ingress to another does need to be planned to avoid losing messages).